docs/data_management/tutorial/globus_individual_tutorial.md

```diff
@@ -0,0 +1,72 @@
+
+# Globus
+
+In this tutorial, we will guide you on how to effectively use Globus for managing and transferring research data. We will explore what Globus is, why you might use it, and walk you through the essential steps to get started. You will also learn how to set up your Globus account, access the platform, find collections shared with you, and search for collections by name.
+
+## Prerequisites
+
+For this tutorial, you will need your BlazerID to authenticate using Single Sign-On (SSO).
+
+## What is Globus, and Why Would You Use It?
+
+Globus is a service that simplifies the process of transferring, sharing, and managing large datasets. It is widely used in research environments where data needs to be moved between different systems, institutions, or storage types.
+
+**Key Features of Globus**:
+
+- It ensures that your data is protected during transfers with encryption.
+- It automatically retries failed transfers, and report the status of your data transfer.
+- It simplifies complex transfer tasks with an intuitive web interface.
+- It handles transfers of all sizes, from small files to multi-terabyte datasets.
+
+## Why You Use Globus?
+
+- Globus is optimized for high-performance environments, reducing transfer times.
+- It has a GUI and is accessible via a web browser, making it easy to initiate transfers from anywhere.
+- Globus allows you to easily share data with collaborators, regardless of their institution.
+
+## How Do You Get Onto Globus?
+
+Getting started with Globus involves a few steps. Here is how you can set up your account and start using the platform:
+
+- To get to the Globus Web App, point your browser to <https://app.globus.org>
+
+    ![first-page-globus](/image/first-page-globus.png)
+
+- Click on **Log in** at the upper right corner.
```

Comment:
As a stylistic choice, I think using "Double Quotes" for GUI elements like buttons and text fields is best, rather than bold. The reason is that they don't appear in bold in the GUI.

Admittedly, they don't appear in quote as well. But, in US English, the double quotes indicate that the material comes from another source. In this case, the source is the GUI elements themselves.

Reply:
Fixed.